Automattic/wp-calypso
# DIFM design picker unwanted sass cleanup

#### Changes proposed in this Pull Request

* This PR cleans up a left over SASS file from #56723

#### Testing instructions
- Go to /start/do-it-for-me 
- Make sure the design picker is not broken
